### ISSUE TITLE
Code for training

### ISSUE THREAD

**Issue by viniciusarasantos** [isarlab-department-engineering/LSVO#3]
Hey, thanks for all the awesome work you've made available here

It has being 4 months since the last update. I'm trying to use some of the ideas you from the paper to do some Visual Odometry using different data (disparity map) extracted from the KITTI dataset. 

I'm trying to transform the input poses for some training. The pose files have dim=(N, 12) and are may be described as the array elements={R1, R2, R3, Tx, R4, R5, R6, Ty, R7, R8, R9, Tz}. 

So for a training task I'm trying to transform this ground truth to a 6-DOF array but I'm not sure if it's I can use "The Robotic Toolbox" directely on this 'elements' array or calculate another transformation array between poses. Can you help me with this?

**Comment by shahabty** [comment]
Hello,
I also need training code in order to work on this project. Could you please release the training dataset and code?

**Comment by gabrielecostante** [comment]
Hi, 
first of all thank you for your interest and sorry for the late response, but, unfortunately, for some reason I did not receive the notification related to this issue. We are currently refactoring the LSVO projects to improve it (the novel version will be Pytorch based). We hope that we will be able to release the code in the next month, so check for updates in this repository.

Thanks again for your interest
Best Regards

**Comment by amiltonwong** [comment]
Great! @gabrielecostante

**Comment by Adamquhao** [comment]
@
We are looking forward to the release of training source code.

**Comment by Adamquhao** [comment]
@gabrielecostante We are looking forward to the release of training source code !!!

**Comment by Adamquhao** [comment]
where？

**Comment by yjhcrossover** [comment]
> Hi, first of all thank you for your interest and sorry for the late response, but, unfortunately, for some reason I did not receive the notification related to this issue. We are currently refactoring the LSVO projects to improve it (the novel version will be Pytorch based). We hope that we will be able to release the code in the next month, so check for updates in this repository.
> 
> Thanks again for your interest Best Regards

Thanks for this repository!how can I find the Pytorch version of this code？thank you in advance！
